qutebrowser/config/configdata.yml

```diff
@@ -3194,3 +3194,22 @@ bindings.commands:
 
     * register: Entered when qutebrowser is waiting for a register name/key for
       commands like `:set-mark`.
+
+## logging
+
+logging.level.ram:
+  default: null
+  type:
+    name: LogLevel
+    none_ok: true
+  desc:
+    Level for in-memory logs.
+
+logging.level.console:
+  default: null
```

Comment:
Does it really make sense for this to be `null` by default (especially since the CLI flags override it)? Why not `info`?